Lib/reprlib.py

```diff
@@ -62,6 +63,27 @@ def repr1(self, x, level):
         else:
             return self.repr_instance(x, level)
 
+    def _join(self, pieces, level):
+        if self.indent is None:
+            return ', '.join(pieces)
+        if not pieces:
+            return ''
+        indent = self.indent
+        if isinstance(indent, int):
+            if indent < 0:
+                raise ValueError(
+                    f'Repr.indent cannot be negative int (was {indent!r})'
+                )
+            indent *= ' '
+        try:
+            sep = ',\n' + (self.maxlevel - level + 1) * indent
+        except TypeError as error:
+            raise TypeError(
+                'Repr.indent must be of type NoneType, str or int, '
+                f'not {type(indent)}'
```

Comment:
As `None` is an interpreter/language-reserved singleton, it is redundant, verbose and unidiomatic to refer to its type as `NoneType` instead of simply `None` in the context of a user-facing error message. This is also how its type is referred to in static type annotations and type checkers. Furthermore, I grepped all the `TypeError`s raised by the Python standard library that have the string `"None"` or `NoneType` somewhere in the first line of the exception text, and all of them referred to it as `None` alongside other types while none referred to it as `NoneType`.

To be shorter and more idiomatic while avoiding this issue entirely, we could just elide "of type" in the message:

```suggestion
                f'Repr.indent must be a str, int or None, not {type(indent)}'
```

Reply:
Huh, fair enough, I guess. :) Thanks!